framework/scenario/tests/contract_without_macros.rs

```diff
@@ -23,7 +22,7 @@ mod module_1 {
     /////////////////////////////////////////////////////////////////////////////////////////////////
     //////// CONTRACT TRAIT /////////////////////////////////////////////////////////////////////////
     /////////////////////////////////////////////////////////////////////////////////////////////////
-    pub trait VersionModule: multiversx_sc::contract_base::ContractBase + Sized {
+    pub trait VersionModule: ContractBase + Sized {
```

Comment:
Leave the fully qualified version in place, this is how it gets generated by the macros